content/news/2024-07-08-galaxy-social/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Galaxy Social - Democratising Galaxy’s social media activity
+date: '2024-07-08'
+tease: A summary of what Galaxy Social is.
+tags: [outreach]
+subsites: [all-eu, global, us]
+main_subsite: eu
+---
+
+
+Galaxy Social is an innovative project aimed at democratizing and streamlining social media activities for the Galaxy community.
+Developed by the Freiburg Galaxy Team, this collaborative tool addresses several limitations of traditional social media management methods.
+
+Key features of Galaxy Social include:
+
+- Democratized Access: Enables community members to contribute and collaborate on social media posts easily.
+- Simple Contribution Process: Uses markdown for creating new posts, making it accessible to a wider range of contributors.
+- Review System: Implements a pull request workflow for peer review and coordination of posts.
+- Multi-Platform Support: Currently supports Bluesky, Mastodon, Matrix, and Slack, with plans to expand to more platforms.
+- Automated Input: Incorporates bots to retrieve content from various feeds, including YouTube and GitHub, with plans to expand for more input bots.
+- Preview Feature: Allows contributors to see how posts will appear before they go live.
+- Customizable Plugins: Written in Python, making it easy to extend functionality.
+
+Galaxy Social addresses common challenges in social media management, such as manual processes, lack of review mechanisms, coordination difficulties, and security risks associated with credential sharing.
+By implementing a GitHub-based workflow, it encourages community participation while maintaining quality control.
+
+Future plans for Galaxy Social include supporting more social media platforms, introducing time-shifting for posts, improving video handling, and refining post-failure management.
+
+For those interested in contributing or learning more about Galaxy Social, visit the [project's GitHub repository](https://github.com/usegalaxy-eu/galaxy-social), [Poster](https://f1000research.com/posters/13-758), and [Slides](https://f1000research.com/slides/13-759).
+
+This innovative approach to social media management not only enhances the Galaxy Project's outreach efforts but also serves as a model for other scientific communities looking to improve their social media presence collaboratively and securely.
```

Comment:
We hope ... and we invite everyone to contribute to ... this github repo .. you see here an example post ... and here the result ....

Reply:
Is the new edit what you had in mind?